src/index.html

```diff
@@ -1,14 +1,33 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header"> <a class="header_img" href="#"> <img src="images/logo.png" alt="logo"> </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"> <a class="is-active nav__link" href="#apple"> Apple </a> </li>
```

Comment:
the space ` ` between tags is not necessary, the same about spaces before and after the text 
```suggestion
          <li class="nav__item"><a class="is-active nav__link" href="#apple">Apple</a></li>
```